files/image_config/warmboot-finalizer/finalize-warmboot.sh

```diff
@@ -76,6 +76,20 @@ function stop_control_plane_assistant()
     fi
 }
 
+function restore_counters_folder()
+{
+    debug "Restoring counters folder after warmboot..."
+
+    modules=("portstat-0" "dropstat" "pfcstat-0" "queuestat-0" "intfstat-0")
+    for module in ${modules[@]}
+    do
+        statfile="/host/$module"
```

Comment:
Same comments as utilities PR. Please move counters to a subfolder under/host.

Reply:
I have updated the code as you reviewed. Thanks a lot.